### ISSUE TITLE
Rename project kramdown-rfc7749?

### ISSUE THREAD

**Issue by hildjj** [cabo/kramdown-rfc2629#22]
https://tools.ietf.org/html/rfc7749 has now obsoleted 2629.

**Comment by cabo** [comment]
Yes, I should have known better than to put an RFC name into the project name...
I'm calling the project kramdown-rfc now, but I won't change the name of the gem for now.
(I still need to think about a better command line interface, until that happens, the command will also retain its old name.)
Actually, kramdown-rfc will move to XML2RFCv3 at some point so naming it after rfc7749 would not last, either.

**Comment by hildjj** [comment]
I agree with your approach.

**Comment by cabo** [comment]
Since 1.6.1, the tool has been named kramdown-rfc (a478406), with some of the guts still saying 2629.
